ios/kmbuild.sh

```diff
@@ -1,7 +1,19 @@
 #!/bin/sh
 
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/../resources/build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+KEYMAN_MAC_BASE_PATH="$KEYMAN_ROOT/mac"
+
 # Include our resource functions; they're pretty useful!
-. ../resources/shellHelperFunctions.sh
+. "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
+
+# This script runs from its own folder
+cd "$(dirname "$THIS_SCRIPT")"
 
 # Please note that this build script (understandably) assumes that it is running on Mac OS X.
 verify_on_mac
```

Comment:
Does the build script faily early if coreutils isn't installed? (maybe prompt user)

Reply:
It doesn't say "hey, install `coreutils`" if that's what you're asking.  You do get a very rapid build failure, though.

Reply:
Will work on this in a future PR